src/graphics/program-lib/chunks/chunk-validation.js

```diff
@@ -26,7 +27,9 @@ const chunkVersions = {
     lightSpecularBlinnPS: CHUNKAPI_1_55,
     lightSpecularPhongPS: CHUNKAPI_1_55,
     normalVertexPS: CHUNKAPI_1_55,
-    startPS: CHUNKAPI_1_55
+    startPS: CHUNKAPI_1_55,
+    refractionCubePS: CHUNKAPI_1_56,
+    refractionDynamicPS: CHUNKAPI_1_56
```

Comment:
we don't have to add newly added chunks. only if we change their API will we add them here.